cli/tests/pcluster/templates/test_cluster_stack/test_security_group_with_restricted_ssh_access/config_restricted_ssh_cidr.yaml

```diff
@@ -0,0 +1,35 @@
+Image:
+  Os: alinux2
+HeadNode:
+  InstanceType: t2.micro
+  Networking:
+    SubnetId: subnet-12345678
+  Ssh:
+    KeyName: ec2-key-name
+    AllowedIps: 1.2.3.4/19
+Scheduling:
+  Scheduler: slurm
+  SlurmQueues:
+    - Name: queue1
+      Networking:
+        SubnetIds:
+          - subnet-12345678
+      ComputeResources:
+        - Name: compute_resource1
+          InstanceType: c5.2xlarge
+LoginNodes:
+  Pools:
+    - Name: login
+      InstanceType: t3.small
+      Count: 1
+      Networking:
+        SubnetIds:
+          - subnet-12345678
+        AdditionalSecurityGroups:
+          - sg-12345678
+      Ssh:
+        KeyName: kn-12345678
+SharedStorage:
+  - MountDir: /opt/shared/efs/managed/1
+    Name: shared-efs-managed-1
+    StorageType: Efs
```

Comment:
Nitpick: here and in the following yaml files, can we remove all the pieces of the config that are not relevant to the test? Or are we reusing these yaml for other tests (e.g. for the security groups for the shared file systems)?

Reply:
Good Catch.
We shouldn't need this details ..I'll remove them and ensure the test still works